docker-compose.yaml

```diff
@@ -19,31 +19,31 @@ x-models:
     warpscript.maxrecursion: 1000
     warpscript.repository.directory: /usr/local/share/warpscript
     warpscript.extension.logEvent: io.warp10.script.ext.logging.LoggingWarpScriptExtension
-  
+
   redis: &redis
       build:
         context: .
         dockerfile: ./images/redis/Dockerfile
 
 services:
   redis-0:
-    image: redis:5
+    image: redis:6
```

Comment:
seems like you are smuggling a redis upgrade, would be worth moving this to a separate commit and mentioning it...

Reply:
The upgrade landed in s3c last week, this docker-compose is just for development